src/core/taylor.rkt

```diff
@@ -1,7 +1,7 @@
 #lang racket
 
 (require math/number-theory)
-(require "../common.rkt" "../programs.rkt" "reduce.rkt")
+(require "../common.rkt" "../programs.rkt" "reduce.rkt" "../syntax/syntax.rkt")
```

Comment:
Stray import?

Reply:
I think I stopped something from re-exporting this package, and that required new imports in a bunch of places. I think re-exporting is kind of bad and I'm moving away from it.